gtsfm/frontend/correspondence_generator/correspondence_generator_base.py

```diff
@@ -0,0 +1,37 @@
+"""Base class for correspondence generators.
+
+Authors: John Lambert
+"""
+
+import abc
+from typing import Dict, List, Tuple
+
+from dask.delayed import Delayed
+import numpy as np
+
+from gtsfm.common.keypoints import Keypoints
+
+
+class CorrespondenceGeneratorBase:
```

Comment:
No run method?

Reply:
Yeah, neither child subclass implements run(), since it's a glorified wrapper around a bunch of create_computation_graph calls